video_mapper.go

```diff
@@ -420,7 +425,16 @@ func getByline(brightcoveVideo map[string]interface{}, publishReference string)
 	return byline
 }
 
-func get(key string, brightcoveVideo map[string]interface{}) (val string, err error) {
+func getFirstPublishedDate(video map[string]interface{}) (result string) {
+	result, err := get("published_at", video)
```

Comment:
Add it as constant. The requirement is that for video we should use value from published_at for first publication date.